ChangeHistory

```diff
@@ -50,6 +51,8 @@ ticket identifier, "G4CMP-nnn", or using the GitHub pull request, "PR-nn".
 2024-05-02  G4CMP-378 : Correct expression for phonon-QP scattering energy.
 2024-05-02  G4CMP-344 : Improve KaplanQP memory usage with mutable buffers.
 
+2024-05-07  G4CMP-404 : Improve charge carriers-phonons scattering.
```

Comment:
This suggests that you branched off of G4CMP-404, but not G4CMP-120 (see my comment above).

Reply:
If I remember correctly, I branched 407 off of 404 very shortly (less than a week) after branching 404 off of develop. I checked and there are no changes from 120 in 407. I also merge develop into 407 a few days ago to fix some the bugs.

Reply:
Done!